src/index.html

```diff
@@ -1,16 +1,62 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, 
+        user-scalable=no, 
+        initial-scale=1.0,
+        maximum-scale=1.0, 
+        minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://www.airbnb.com.ua/help/search?"
+      method="get"
+      data-qa="big"
+      class="Search"
+    >
+      <label class="SearchBar SearchBar--big">
+        <img class="SearchBar__icon"
+          src="images/Search.svg" 
+          alt="SearchBar"
+        >
+        <input
+          class="SearchBar__input"
+          type="text"
+          name="q"
```

Comment:
what does `q` mean?

Reply:
I decided not to leave the form `action ` empty and added a search on the Airbnb website, q is the field's name when sending the form. 
Example [https://www.airbnb.com.ua/help/search?q=Los+Angeles](). 
Remove that action from forms?

Reply:
just choose a more informative name. `firstQuery`, for example